### PR TITLE
Adjusted CSP in httpd.conf to suit ACE javascript editor

### PR DESCRIPTION
**Pull request by craigcomstock**
Ticket: ENT-12010
Changelog: title

together
https://github.com/cfengine/mission-portal/pull/2563
https://github.com/cfengine/masterfiles/pull/2939

### CODE REVIEW

File: cfe_internal/enterprise/templates/httpd.conf.mustache [hunk 1]
```diff
@@ -215,14 +215,14 @@ LogLevel warn
     object-src 'none'; \
     frame-src 'self'; \
     child-src 'self'; \
-    img-src 'self' avatars.githubusercontent.com badges.gitter.im fonts.gstatic.com kiwiirc.com raw.githubusercontent.com; \
+    img-src 'self' avatars.githubusercontent.com badges.gitter.im fonts.gstatic.com kiwiirc.com raw.githubusercontent.com data:; \
```
Comment (nickanderson) [inline cfe_internal/enterprise/templates/httpd.conf.mustache:218]: @craig, this `data:` is already changed in master. It's after self before avitars. Maybe rebase?

Comment (craigcomstock) [reply cfe_internal/enterprise/templates/httpd.conf.mustache:218]: sure, that's fine...

File: cfe_internal/enterprise/templates/httpd.conf.mustache [hunk 2]
```diff
@@ -215,14 +215,14 @@ LogLevel warn
...
     font-src 'self' data: fonts.googleapis.com fonts.gstatic.com; \
     connect-src 'self' fonts.gstatic.com fonts.googleapis.com; \
     manifest-src 'self'; \
     base-uri 'self'; \
     form-action 'self'; \
     media-src 'self'; \
-    worker-src 'self';"
+    worker-src 'self' blob:"
```
Comment (nickanderson) [inline cfe_internal/enterprise/templates/httpd.conf.mustache:225]: I added this change to my running hub and at least things did not break obviously.

Comment (craigcomstock) [reply cfe_internal/enterprise/templates/httpd.conf.mustache:225]: right, look in js console in host info cmdb show json and policy analyzer, that's where we use ACE editor.

**Review by nickanderson — APPROVED** [review]
This didn't blow things up yesterday when I tried it, so should be OK.

File: cfe_internal/enterprise/templates/httpd.conf.mustache [hunk 3]
```diff
@@ -222,7 +222,7 @@ LogLevel warn
...
```
Comment (craigcomstock) [inline cfe_internal/enterprise/templates/httpd.conf.mustache:225]: @nickanderson asked me why no `;` at the end and I knew this worked but was curious.

https://www.w3.org/TR/CSP/

> A serialized CSP is an [ASCII string](https://infra.spec.whatwg.org/#ascii-string) consisting of a semicolon-delimited series of [serialized directives](https://www.w3.org/TR/CSP/#serialized-directive), adhering to the following ABNF grammar [[RFC5234]](https://www.w3.org/TR/CSP/#biblio-rfc5234):
serialized-policy =
    serialized-directive *( optional-ascii-whitespace ";" [ optional-ascii-whitespace serialized-directive ] )

**Pull request merged**